src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,23 @@
 package core.mate.academy.model;
 
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
-public class Truck extends Machine {
+public class Truck extends Machine implements MachineProducer<Truck> {
     public Truck() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Truck started to work");
     }
+
+    @Override
+    public List<Truck> get() {
+        return List.of(new Truck(), new Truck());
+    }
 }
```

Comment:
no need to implement from MachineProducer
instead, you should create separate package `producer`, and create Producers for each model, implemented MachineProducer<?> interface
also, according to the task, you should add some fields to this model
![image](https://github.com/mate-academy/jv-pecs/assets/73041091/e96a6970-9bac-40b2-864e-3d1b13d1e9be)
```suggestion
public class Truck extends Machine {
    public Truck() {
    }

    @Override
    public void doWork() {
        System.out.println("Truck started to work");
    }
}
```